schema.graphql

```diff
@@ -82,7 +82,7 @@ type BuyEvent @entity {
 type StateChange @entity {
   id: ID! # tx
   timestamp: BigInt!
-  changes: [String!]!
+  changes: [String!]! #call event logs or something
```

Comment:
Dude, I recon you just change this now. It will be quick. otherwise we will build things on top of this with the wrong name and need to change it in 20 repos...